resources/frontend_client/index_template.html

```diff
@@ -19,16 +19,18 @@
         <main class="Main flex flex-column flex-full" ng-view></main>
     </body>
 
-
-    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js"></script>
-    <script>
-        WebFont.load({
-          google: { families: ['Lato:n3,n4,n7'] }
-        });
+    <script type="text/javascript">
+     (function loadSrc(src, onload) {
+         var script = document.createElement('script');
+         script.type  = "text/javascript";
+         script.async = true;
+         script.src   = src;
+         if (onload) script.onload = onload;
+         document.body.appendChild(script);
+         return loadSrc;
+     })('https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js', 'WebFont.load({google:{families:["Lato:n3,n4,n7"]}});')('https://maps.googleapis.com/maps/api/js', null);
```

Comment:
Also the minified GA code below this does essentially the same thing, plus some GA stuff. We could merge them. Honestly the minification isn't going to make a bit of difference considering how much JS we load for the main app, but if we really wanted to we could put this in a separate file and minify it.

```
(function(w, d) {
    function loadScript(src, onLoad) {
        var s = d.createElement("script");
        s.async = true;
        s.src = src;
        s.onLoad = onLoad;
        var m = d.getElementsByTagName("script")[0];
        m.parentNode.insertBefore(s, m);
    }

    // Google WebFont
    loadScript("https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js", function() {
        WebFont.load({ google: { families: ["Lato:n3,n4,n7"] } });
    });

    // Google Maps
    loadScript("https://maps.googleapis.com/maps/api/js");

    // Google Analytics
    w.GoogleAnalyticsObject = "ga";
    w.ga = w.ga || function() { (w.ga.q = w.ga.q || []).push(arguments) };
    w.ga.l = 1*new Date();
    loadScript("https://www.google-analytics.com/analytics.js");
})(window, document);
```